### DISCUSSION TITLE
How to direct rdkit install to go to a different python on my system?

### DISCUSSION THREAD

**Discussion by BastiaanBergman** [Homebrew/discussions#2568] [Casks] [answered]
`brew install rdkit`
works beautifully, however it installs rdkit in:
`/usr/local/lib/python3.9/site-packages`

instead of where I want it:
`/Users/bastiaanbergman/anaconda3/lib/python3.8/site-packages/`

Is there a simple thing I can do to make sure it goes to the correct environment?

My bottom line problem is that in my desired environment I get this error:
`>>> import rdkit`
`Traceback (most recent call last):`
 ` File "<stdin>", line 1, in <module>`
`ModuleNotFoundError: No module named 'rdkit'`

In the other (not desired) environment rdkit is working fine.

**Comment by SMillerDev** [comment]
No, anaconda and brew are two separate package managers. Brew can't install with anaconda dependencies or the other way around.

**Comment by carlocab** [comment, marked as answer]
You can try adding `/usr/local/lib/python3.9/site-packages` to your `PYTHONPATH`. This may or may not work, and even if it seems like it does, you might encounter weird bugs.

If you really must use Anaconda Python 3.8, you can create your own tap (see `brew help tap-new`) and use `brew extract` to create your own version of `rdkit` that uses the Python that you want. This will take a bit of work if you're not already familiar with how formulae work, though, and you'll need to build your own `rdkit` from source. Reading the docs will help a lot.
